NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -38,14 +38,21 @@ let SkipTimeInterval: Double = 15
     private let sleepTimerDefaultAccessibilityLabel = NSLocalizedString("Sleep Timer", bundle: Bundle.audiobookToolkit()!, value: "Sleep Timer", comment:"Sleep Timer")
 
     private var audiobookProgressView = DownloadProgressView()
-
+    
     private let chapterInfoStack = ChapterInfoStack()
     public var coverView: AudiobookCoverImageView = { () -> AudiobookCoverImageView in
         let image = UIImage(named: "example_cover", in: Bundle.audiobookToolkit(), compatibleWith: nil)
         let imageView = AudiobookCoverImageView.init(image: image)
         return imageView
     }()
-    private let seekBar = ScrubberView()
+    private let seekBar: ScrubberView = {
+        if #available(iOS 12.0, *),
+           UIScreen.main.traitCollection.userInterfaceStyle == .dark {
+            return ScrubberView(tintColor: NYPLColor.actionColor)
+        } else {
+            return ScrubberView()
+        }
+    }()
```

Comment:
once the scrubber view is initialized, does the tintColor change if the user switches from dark to light and vice versa?

Reply:
np.... feel free to do it in a separate PR if that's easier